packages/gatsby-theme-docs/configure-theme.js

```diff
@@ -0,0 +1,25 @@
+const pkg = require('./package.json');
+
+function configureThemeWithAddOns(themeOptions = {}) {
+  const { addOnPlugins, ...coreThemeOptions } = themeOptions;
+
+  if (!addOnPlugins || !Array.isArray(addOnPlugins)) {
+    throw new Error(
+      'Missing or wrong format for "addOnPlugins" option. Expected a list of Gatsby theme add-on plugins.'
+    );
+  }
```

Comment:
Does this imply that gatsby-theme-docs cannot be used as. a standalone?

Reply:
No you can still use that. This is just an "helper function" to keep the configuration simple.

See detailed explanation: https://github.com/commercetools/commercetools-docs-kit/blob/2f6c3bf8b24ba6e81a456b5fc9e1d1db323bbf47/packages/gatsby-theme-docs/README.md#using-theme-with-add-ons